.gitignore

```diff
@@ -10,6 +10,7 @@
 *.exe
 mongo-c-performance
 performance-testdata/*
+!performance-testdata/README.md
```

Comment:
The script to download test data, `download-test-data.py` expects this directory to exist. I added a README.md to the directory so it is tracked in git.